api/openapi-spec/swagger.json

```diff
@@ -19727,6 +19727,26 @@
      }
     }
    },
+   "v1alpha1.SnapshotableSourceTypeVolumesLists": {
+    "description": "SnapshotableSourceTypeVolumesLists includes the list of volumes which were included in the snapshot and volumes which were excluded from the snapshot",
```

Comment:
I think it might be more straightforward to always list all volumes whether they are snapshottable or not.  This way, a user can quickly see which volumes can be restored and which can not without needing to apply extra logic to understand that volumes which are not listed are implicitly excluded.  What do you think?  If we adopt this change it would also make naming this field easier (ie. SnapshotVolumeLists

Reply:
I thought when we talked about this that we shouldn't put in this list any volume which is not datavolume/pvc, should we add container disks here too? cloud init? seems irrelevant, but sure I can do it

Reply:
I will put all the volumes in the lists for completeness. It will sure make the field name easier :)

Reply:
This looks great now!